packages/terra-time-input/docs/README.md

```diff
@@ -2,7 +2,15 @@
 
 The terra-time-input component is a controlled input component for entering time. It is a controlled component because it manages the state of the value in the input. Because this is a controlled input component, it cannot accept the defaultValue prop as it always uses the value prop. React does not allow having both the defaultValue and value props.
 
-The currently supported time format is the 24-hour format (hh:mm). The time input enforces the entry that masks to the format. The hour input only accepts values between 00 and 23 and the minute input only accepts values between 00 and 59. For example, a time of 25:65 cannot be entered. A 0 will automatically be prepended to the hour if the entered hour is greater than 2. Likewise, a 0 will automatically be prepended to the minute if the entered minute is greater than 5.
+The currently supported time format is the 24-hour format (hh:mm). The time input enforces the entry that masks to the format. The hour input only accepts values between 00 and 23 and the minute input only accepts values between 00 and 59. For example, a time of 25:65 cannot be entered. Any times that are not in the format hh:mm will be ignored when initialized a terra-time-input component.
+
+There are two types of clocks that are currently supported: A 12-hour clock and a 24-hour clock. This can be controlled through the `variant` prop on the component.
+
+If a 24 hour clock is chosen, the hour displayed will be a value between 0 and 23 inclusively, and the minute displayed will be between 0 and 59 inclusively as well. A 0 will automatically be prepended to the hour if the entered hour is greater than 2. Likewise, a 0 will automatically be prepended to the minute if the entered minute is greater than 5. In addition, the inputs can be changed through the use of the up and down arrow keys on a non-mobile display.
```

Comment:
This wording is a bit confusing to me.

> A 0 will automatically be prepended to the hour if the entered hour is greater than 2. 

For example, I would expect when entering '1' then tabbing to result in '01', but this reads like only would only occur for entering '3-9' then tabbing to result in '03 - 09'.

Then,

> Likewise, a 0 will automatically be prepended to the minute if the entered minute is greater than 5.

I would expect any value entered '0-9' then tabbing to result in '00-09', but to me this reads time input would  only occur for entering '6-9' then tabbing to result in '06 - 09'.

Reply:
So for these inputs, pressing a single key character will actually append the 0 if the value is between 2 and 9 for the hour on 23 clocks. Likewise, this same behavior will occur for minutes where the key press is between 5 and 9. Tabbing behavior will only occur for when you type in a 0, 1, or 2.

I can see where the confusion lies, and it's in the fact I didn't specify a single key press. Try this doc out and see if it sounds better. https://github.com/cerner/terra-core/pull/1027/commits/7e58ae016b13c4c56b9d2e1a5c5a7b2434f77986